lib/trino-hive-formats/src/main/java/io/trino/hive/formats/TrinoDataInputStream.java

```diff
@@ -230,10 +232,14 @@ public int read()
     public long skip(long length)
             throws IOException
     {
+        if (length <= 0) {
+            return 0;
```

Comment:
for length == 0, it's obvious.
why also for length < 0? is it worth explanatory comment?

Reply:
This is the behavior specified in the Javadoc and in the default implementation.